vscode/BUILD

```diff
@@ -32,3 +32,15 @@ sh_binary(
         ":vscode-sources",
     ],
 )
+
+sh_binary(
+    name = "packager",
+    srcs = ["packager.sh"],
+    data = [
+        ":language-configuration.json",
+        ":dataform_logo.png",
```

Comment:
Is this from an existing file in repo?

If so, rather than copying the file, you can use the bazel rule in tools/common to copy the file from where it was to here

Reply:
It is not, I had to make a logo that was the correct size